testnets/xiontestnet/chain.json

```diff
@@ -0,0 +1,135 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "xion-testnet-1",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Xion Testnet",
+  "chain_id": "xion-testnet-1",
+  "bech32_prefix": "xion",
+  "daemon_name": "xiond",
+  "node_home": "$HOME/.xiond",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uxion",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uxion"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/burnt-labs/xion",
+    "recommended_version": "v0.3.2",
+    "compatible_versions": [
+      "v0.1.0",
```

Comment:
is v0.1.0 really compatible (non-consensus-breaking) with v0.3.2? I'm guessing it's not. But if they are, they should be recorded as a single version object in the versions array.

Reply:
Nice catch. Removed. Thanks!